core/modules/field/modules/options/js/options.js

```diff
@@ -562,7 +562,7 @@ Backdrop.optionsElement.prototype.optionsToText = function() {
   var $rows = $('tbody tr', this.optionsElement);
   var output = '';
   var inGroup = false;
-  var rowCount = $rows.size();
+  var rowCount = $rows.length;
```

Comment:
This still works as expected when toggling between the drag-drop UI vs. the manual/textual version 👍🏼